### DISCUSSION TITLE
Unreal Engine 5.1 support?

### DISCUSSION THREAD

**Discussion by tstkenny** [Sixze/ALS-Refactored#184] [Q&A]
Is there any plan for UE 5.1?  Thank you.

**Comment by oivio** [comment]
> Is there any plan for UE 5.1? Thank you.

I did not have any problems running it on UE5.1
Probably you will have to rebuild the Binaries possibly with Visual Studio but that's all.

**Comment by Alex-G** [comment]
Jep,
I just compiled it, you get a message that it is made for UE 5.0 not UE 5.1.
You can edit `ALS.uplugin` to get rid of this message.
`"EngineVersion": "5.0.0"` -> `"EngineVersion": "5.1.0"`
![image](https://user-images.githubusercontent.com/5215249/202850020-5d83ee69-60fe-45f1-b5a4-8ca63643383a.png)
